.travis.yml

```diff
@@ -5,8 +5,12 @@ services:
 cache:
   directories:
   - .glide
-script:
-- make verify build test images
+jobs:
+  include:
+    - stage: Verify and Build
+      script: make verify build
+    - stage: Test and Make Image
+      script: make test images
```

Comment:
interesting, I didn't realize that. So, then in the current form of the PR the 2nd stage will not use the output from the 1st, right?  While I'm not overly concerned about the time of the travis build, it seems a bit wasteful to build everything twice just to see the nice grouping that "stage" provides. Especially, when in most cases I would expect people to always just scroll to the bottom of the console output to see the error message.

Seems like "stage" would be most useful when you have totally independent tasks that do not need to share local storage - unfortunately we do share it.